gitlab/gitlab_exporter.py

```diff
@@ -0,0 +1,127 @@
+import requests
+import json 
+import sys
+
+GITLAB_URL = "https://gitlab.com/api/v4" 
+PORT_API_URL = "http://localhost:3000/v1" # TODO: change to "https://api.getport.io/v1"
+WEBHOOK_URL = "https://smee.getport.io/SNzcBJlHUFfzHDO" 
+
+PORT_CLIENT_ID = sys.argv[1]
+PORT_CLIENT_SECRET = sys.argv[2] 
+GITLAB_API_TOKEN = sys.argv[3]
+GROUP_ID = sys.argv[4]
+
+def create_webhook():
+    api_url = f"{GITLAB_URL}/groups/{GROUP_ID}/hooks"
```

Comment:
The core api_url (up until {GROUP_ID}) can be made a const variable, and then in the functions that you need it you can do:

`hooks_url = f"{API_URL}/hooks"`
`projects_url = f"{API_URL}/projects"`

and so on